config/permissions.yml

```diff
@@ -1161,6 +1161,11 @@ feature_new_statement_form:
     label: 'Stell. verf. Formular anzeigen'
     loginRequired: true
     parent: feature_statement
+feature_show_notice_draft_statement:
+    description: 'some project with Gateway as a project type (diplanbeteiligung) should not be show not for submitter in statement public'
```

Comment:
```suggestion
    description: 'some project with Gateway as a project type (diplanbeteiligung) should not be shown for the submitter in the public statement'
```

Reply:
done in [c2a3d98](https://github.com/demos-europe/demosplan-core/pull/2721/commits/c2a3d983f432f752a9db3688255f13d9eefdf306)